system/mount.py

```diff
@@ -358,13 +358,17 @@ def main():
                     res,msg = mount(module, **args)
             elif 'bind' in args.get('opts', []):
                 changed = True
-                cmd = 'mount -l'
+                cmd = 'findmnt -rn %s' % args['name']
```

Comment:
we provide such a function in basic.py `module.get_bin_path('<executable>', required=false, opt_dirs=[])`

use `required=True` to automatically fail the module if the binary is not found.